src/main/scala/ksql/consumer/deserializer/VoteDeserializer.scala

```diff
@@ -0,0 +1,22 @@
+package ksql.consumer.deserializer
+
+import org.apache.kafka.common.serialization.{Deserializer, StringDeserializer}
+
+import io.circe.parser._
+
+import ksql.models.Vote
+
+class VoteDeserializer extends Deserializer[Vote] {
```

Comment:
I think we don't need dedicated serialization, we can use kafka default StringSerializer and StringDeserializer